src/Forms/Forms.Component.js

```diff
@@ -102,7 +102,7 @@ export const FormsComponent = () => {
                     is displayed below the input field.`}
                 title='Input Help Elements'>
                 <>
-                    <FormSet>
+                    <FormGroup>
```

Comment:
Could these be replaced with `<FormFieldSet` instead of `<FormGroup`?

Reply:
Do they need to be a FormFieldSet?

Reply:
nope!  I didn't mean to suggest they should be. I just read the PR description and saw
> It was replaced by FormFieldset

So I thought I'd ask